easybuild/easyconfigs/p/pigz/pigz-2.3.1-goolf-1.4.10.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'pigz'
+version = '2.3.1' 
+
+homepage = 'http://zlib.net/pigz/'
+description = """ pigz, which stands for parallel implementation of gzip, is a fully 
+ functional replacement for gzip that exploits multiple processors and multiple cores 
+ to the hilt when compressing data. pigz was written by Mark Adler, and uses the zlib 
+ and pthread libraries. """
```

Comment:
yeap, you are right, that was me about the indentation. just keep doing what you do ;-)